wazo_agid/modules/tests/test_call_recording.py

```diff
@@ -208,28 +245,11 @@ def test_record_caller_records_when_not_an_outcall_and_all_enabled(
 
         record_caller(self.agi, self.cursor, None)
 
-        calls = [call('WAZO_CALL_RECORD_ACTIVE'), call('WAZO_USERUUID'), call('XIVO_OUTCALLID')]
-        self.agi.get_variable.assert_has_calls(calls)
-
-        start_mix_monitor.assert_called_once()
-
-    @patch('wazo_agid.modules.call_recording._start_mix_monitor')
-    @patch('wazo_agid.objects.User')
-    def test_record_caller_records_when_an_outcall_and_all_enabled(self, user, start_mix_monitor):
-        agi_variables = {
-            'WAZO_CALL_RECORD_ACTIVE': '',
-            'WAZO_USERUUID': 'the-users-uuid',
-            'XIVO_OUTCALLID': '1',
-        }
-        self.agi.get_variable.side_effect = agi_variables.get
-        user.return_value = Mock(
-            call_record_outgoing_internal_enabled=True,
-            call_record_outgoing_external_enabled=True,
-        )
-
-        record_caller(self.agi, self.cursor, None)
```

Comment:
Why removing test on black commit?

Reply:
It's `black` and `flake8`. When adding them `flake8` correctly identified this test to be a duplicate, so I removed it. Not sure why it was there twice. Maybe a remnant of a merge conflict or just an accident.

Reply:
Oh weird ..
Thank you :+1: